src/plugins/discover/public/application/components/sidebar/string_progress_bar.tsx

```diff
@@ -17,35 +17,24 @@
  * under the License.
  */
 import React from 'react';
-import { EuiFlexGroup, EuiFlexItem, EuiProgress, EuiText, EuiToolTip } from '@elastic/eui';
+import { EuiProgress, EuiToolTip } from '@elastic/eui';
 
 interface Props {
   percent: number;
   count: number;
+  value: string;
 }
 
-export function StringFieldProgressBar(props: Props) {
+export function StringFieldProgressBar({ value, percent, count }: Props) {
+  const tooltipContent = `${value} ${count}`;
   return (
     <EuiToolTip
       anchorClassName="dscProgressBarTooltip__anchor"
-      content={props.count}
+      content={tooltipContent}
       delay="regular"
       position="right"
     >
```

Comment:
Do you think it might make sense to cover more elements with the Tooltip? Since the graph is now smaller (which is nice, less vertical screenspace), it could e.g. surround 

<img width="136" alt="Bildschirmfoto 2020-06-18 um 17 41 33" src="https://user-images.githubusercontent.com/463851/85042247-45552580-b18b-11ea-8d66-ebc5ff8d303d.png">

And it could provide more information like "Value: ${value}, count: ${count} (${percent})"

CC @myasonik, for a11y input